lib/client-api/src/story_store.ts

```diff
@@ -157,11 +157,11 @@ export default class StoryStore {
     this.pushToManager();
     if (this._channel) this._channel.emit(Events.RENDER_CURRENT_STORY);
 
-    const storyIds = Object.keys(this._stories);
-    if (storyIds.length) {
+    const [firstStory] = Object.values(this._stories);
+    if (firstStory) {
       const {
         parameters: { globalArgs: initialGlobalArgs, globalArgTypes },
-      } = this.fromId(storyIds[0]);
+      } = this.mergeAdditionalDataToStory(firstStory);
```

Comment:
@shilman this is a separate issue, but am I right in saying this code should really just read:

```js
const { globalArgs: initialGlobalArgs, globalArgTypes } = this._globalMetadata.parameters;
```

Right? We've guaranteed that globalArg stuff is in the global parameters already, right?

Reply:
I think, the answer is yes. Because `this.combineStoryParameters` is already merging `this._globalMetadata.parameters` into story parameters.

Reply:
@tmeasday yes, this should only be set in the global metadata